rpc/swap.go

```diff
@@ -120,15 +120,17 @@ func (s *SwapService) GetPast(_ *http.Request, req *GetPastRequest, resp *GetPas
 
 // OngoingSwap represents an ongoing swap returned by swap_getOngoing.
 type OngoingSwap struct {
-	ID             types.Hash          `json:"id" validate:"required"`
-	Provided       coins.ProvidesCoin  `json:"provided" validate:"required"`
-	ProvidedAmount *apd.Decimal        `json:"providedAmount" validate:"required"`
-	ExpectedAmount *apd.Decimal        `json:"expectedAmount" validate:"required"`
-	ExchangeRate   *coins.ExchangeRate `json:"exchangeRate" validate:"required"`
-	Status         types.Status        `json:"status" validate:"required"`
-	StartTime      time.Time           `json:"startTime" validate:"required"`
-	Timeout0       *time.Time          `json:"timeout0"`
-	Timeout1       *time.Time          `json:"timeout1"`
+	ID                        types.Hash          `json:"id" validate:"required"`
+	Provided                  coins.ProvidesCoin  `json:"provided" validate:"required"`
+	ProvidedAmount            *apd.Decimal        `json:"providedAmount" validate:"required"`
+	ExpectedAmount            *apd.Decimal        `json:"expectedAmount" validate:"required"`
+	ExchangeRate              *coins.ExchangeRate `json:"exchangeRate" validate:"required"`
+	Status                    types.Status        `json:"status" validate:"required"`
+	LastStatusUpdateTime      time.Time           `json:"lastStatusUpdateTime" validate:"required"`
+	StartTime                 time.Time           `json:"startTime" validate:"required"`
+	Timeout0                  *time.Time          `json:"timeout0"`
+	Timeout1                  *time.Time          `json:"timeout1"`
+	EstimatedTimeToCompletion time.Duration       `json:"estimatedTimeToCompletion" validate:"required"`
```

Comment:
If you are looking to trim a few characters you could use `EstimatedTimeRemaining`.  (No need to make that change unless you like it, it only save 3 symbols.)